src/components/Header/Header.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ReactComponent as ShoppingCartIcon } from '../../images/shopping-cart.svg';
+import logo from '../../images/logo.png';
+
+import {
+  HeaderMain, Image, Name, Search, ButtonCart,
+} from './Header.styled';
+
+function Header() {
+  return (
+    <HeaderMain isYellow>
+      <Image src={logo} alt="logo" />
+      <Name>
+        Furny
+      </Name>
+      <Search type="text" placeholder="Search" />
```

Comment:
InputSearch ....what do you think? i consider that InputSearch add more semantic

Reply:
I agree.